src/components/Card.test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Card from './Card';
+import { shallow } from 'enzyme';
+
+
+describe('Card', () => {
+
+  test('shallow mount', () => {
+    const card = shallow(
+      <Card deleteCallback={()=>{}} />
```

Comment:
We could potentially have a few more tests here. Specifically, I'm thinking about tests which verify the render logic as it relates to the prop values.

This means, we might have one snapshot test where we provide the `text` prop, and then both verify that it matches the snapshot we previously took and also that we can find an `h1` tag with that text content in it.

We could then also do a similar test for when the `emoji` prop is provided, and finally one for when both props are provided.